R/add_rasterly_heatmap.R

```diff
@@ -15,6 +16,7 @@
 #' @examples
 #' \dontrun{
 #'if(requireNamespace("plotly") && requireNamespace("data.table")) {
+#'  library(lubridate)
```

Comment:
It is mainly used to handle the time. Well, I should call 
```
if(requireNamespace("plotly") && requireNamespace("data.table") && requireNamespace("lubridate"))
```
Instead. 

It will be fixed in the new PR

Reply:
Perfect, no need to open a new PR, can just make the change here.